wstool/WorkingSet.cpp

```diff
@@ -0,0 +1,91 @@
+#include "WorkingSet.h"
+
+using namespace std;
+
+namespace WorkingSet
+{
+    int64_t timing = 0;
+    map<int64_t, string> deAlias;
+    map<string, vector<int64_t>> virAddr;
```

Comment:
Unless you have a good reason (this isn't one) you should avoid using strings as keys. Make it either a struct or choose a primative